packages/readmodel/test/eserviceTemplateAggregator.test.ts

```diff
@@ -20,8 +20,12 @@ describe("E-service template aggregator", () => {
     const certifiedAttribute = getMockEServiceAttribute();
     const doc = getMockDocument();
     const interfaceDoc = getMockDocument();
-    const riskAnalysis1 = getMockValidRiskAnalysis(tenantKind.PA);
-    const riskAnalysis2 = getMockValidRiskAnalysis(tenantKind.PRIVATE);
+    const riskAnalysis1 = getMockValidEServiceTemplateRiskAnalysis(
+      tenantKind.PA
+    );
+    const riskAnalysis2 = getMockValidEServiceTemplateRiskAnalysis(
+      tenantKind.PRIVATE
+    );
```

Comment:
if these const is not used in more points the test you can skip this declaration and using `getMockValidEServiceTemplateRiskAnalysis` directly in EserviceTemplate creation in line [55](https://github.com/pagopa/interop-be-monorepo/pull/1998/files#diff-0066d29c37c0d42b4bcfc21824e96e8ac4a295528648f1f3523afc9819c5b708R55).